tests/python/pants_test/backend/jvm/tasks/jvm_compile/java/test_zinc_compile_integration.py

```diff
@@ -373,7 +373,7 @@ def main(args: Array[String]) {
             path = os.path.join(compile_dir, path_suffix)
             self.assertTrue(os.path.exists(path), "Want path {} to exist".format(path))
 
-  def test_hermetic_binary_with_3rdparty_dependencies(self):
+  def test_hermetic_binary_with_3rdparty_dependencies_ivy(self):
```

Comment:
In case the default becomes coursier, could you please explicitly set the resolver to ivy in the config?